### ISSUE TITLE
Release 0.3 with rubyzip 1.0 support

### ISSUE THREAD

**Issue by rywall** [toretore/zippy#11]
That would be awesome. Thanks!

**Comment by kreba** [comment]
Thanks for the useful gem. We've been using it in major projects for quite a while now. :thumbsup:
A new release with the current fixes would be much appreciated, here too!

**Comment by tboyko** [comment]
Yes, please release the current codebase in the repository to ruby gems.
